source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidTaskUtils.cs

```diff
@@ -9,7 +9,7 @@ class TaskOnCompleteProxy<T> : AndroidJavaProxy
     {
         private Action<T> mCallback;
 
-        public TaskOnCompleteProxy(Action<T> callback) 
+        public TaskOnCompleteProxy(Action<T> callback)
```

Comment:
We can make those classes internal to AndroidTaskUtils as they shouldn't be used directly
For TaskOnCompleteProxy<AndroidJavaObject> and TaskOnSuccessProxy<AndroidJavaObject> we can try to overload those cases and add using for object around callback.